modules/identifying-the-differences-in-allowed-connections-between-project-versions.adoc

```diff
@@ -1,32 +1,83 @@
 
 // Module included in the following assemblies:
 //
-// * operating/manage-network-policies.adoc
+// * operating/build-time-network-policy-tools.adoc
 :_content-type: PROCEDURE
 [id="identifying-the-differences-in-allowed-connections-between-project-versions_{context}"]
 = Identifying the differences in allowed connections between project versions
 
+:FeatureName: Build-time network policy connectivity difference
+include::snippets/technology-preview.adoc[]
+
 Based on the workload and Kubernetes network policy manifests present in each version directory, you can list the differences in allowed connections between two versions of a project. You can view the semantic differences that might not be immediately apparent in a syntactic difference output.
 
+To produce a connectivity difference report, the `roxctl netpol connectivity diff` command requires two folders, `dir1` and `dir2`, each containing Kubernetes manifests, including network policies.
+
+
 .Procedure
 
 * Run the following command to determine the connectivity differences between the Kubernetes manifests in the specified directories:
 +
 [source,terminal,subs="+quotes"]
 ----
-$ roxctl netpol connectivity diff --dir1=_<folder1>_ --dir2=_<folder2>_     <1>
+$ roxctl netpol connectivity diff --dir1=_<folder1>_ --dir2=_<folder2>_ <1>
 ----
 +
-<1> Specify the actual path to the directories containing the Kubernetes manifests you want to compare.
+<1> Specify the actual path to the directories containing the Kubernetes manifests you want to compare. For example, `<folder1>` is `roxctl/netpol/connectivity/diff/testdata/netpol-analysis-example-minimal/` and `<folder2>` is `roxctl/netpol/connectivity/diff/testdata/netpol-diff-example-minimal`.
 +
 [NOTE]
 ====
 The command considers all YAML files that you can accept using `kubectl apply -f`, and then these become valid inputs for your `roxctl netpol connectivity diff` command.
 ====
 +
+.Example output
++
+[source,terminal,subs="+quotes"]
+----
+Connectivity diff:
+diff-type: changed, source: default/frontend[Deployment], destination: default/backend[Deployment], dir1:  TCP 9090, dir2: TCP 9090,UDP 53
+diff-type: added, source: 0.0.0.0-255.255.255.255, destination: default/backend[Deployment], dir1:  No Connections, dir2: TCP 9090
+----
++
 The semantic difference report gives you an overview of the connections that were changed, added, or removed in `dir2` compared to the connections allowed in `dir1`. When you review the output, each line represents one allowed connection that was added, removed, or changed in `dir2` compared to `dir1`.
 If applicable, the `workloads-diff-info` provides additional details about the workloads associated with the added or removed connections.
 
+The `roxctl netpol connectivity diff` command supports the following options:
```

Comment:
Same comments here for this content, re: breaking it out into its own `:_content-type: REFERENCE` module, adding `[options="header"]` to the table, and marking up the options with backticks.